src/lib/y2partitioner/dialogs/popup.rb

```diff
@@ -26,6 +26,22 @@ module Y2Partitioner
   module Dialogs
     # Adapt CWM dialog to allow popup dialogs
     class Popup < CWM::Dialog
+      DEFAULT_MIN_WIDTH = 50
+      private_constant :DEFAULT_MIN_WIDTH
+
+      DEFAULT_MIN_HEIGHT = 18
+      private_constant :DEFAULT_MIN_HEIGHT
+
+      # @!method min_width=(value)
+      #   Sets the popup min width
+      #   @param value [Integer]
+      attr_writer :min_width
```

Comment:
Why creating an accessor instead of redefining a `min_width` method in the subclasses? Just as it's done for `ok_button_label` for example.

Reply:
In fact, the first implementation was a method, but I am not sure, I see these values as a kind of configuration but not a different behavior. I mean, I would not define a new subclass only because these values.

Reply:
That's exactly the CWM approach. Subclasses for redefining mostly everything. Why should this be different?